app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, foreign_key: :author_id, class_name: :User
+  belongs_to :post
+  after_save :update_post_comments_counter
+
+  def update_post_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- **[OPTIONAL]** We can improve this set up by making the `update_post_comments_counter` method private.

  ```
  after_save :update_post_comments_counter

  private

  def update_post_comments_counter
    post.increment!(:comments_counter)
  end
  ```